Sources/CLibMongoC/include/CLibMongoC_bson-config.h

```diff
@@ -128,7 +128,7 @@
 /*
  * Define to 1 if you want extra aligned types in libbson
  */
-#define BSON_EXTRA_ALIGN 1
+#define BSON_EXTRA_ALIGN 0
```

Comment:
do we need to define this here also, or is the one in `etc/generated_headers` sufficient?

Reply:
The change in `etc/generated_headers` produces this output, so both are required

Reply:
oh, right :facepalm: